packages/medusa-plugin-auth/.gitignore

```diff
@@ -9,7 +9,7 @@
 /services
 /loaders
 /auth-strategies
-index.*
+# index.*
```

Comment:
any reason why it is commented now?

Reply:
It was commented when I started working on it, so I uncommented it and than commented it back.